src/jdk.internal.jvmstat/share/classes/sun/jvmstat/perfdata/monitor/AliasFileParser.java

```diff
@@ -143,7 +142,7 @@ public void parse(Map<String, ArrayList<String>> map) throws SyntaxException, IO
                 match(StreamTokenizer.TT_WORD);
 
             } while ((currentToken.ttype != StreamTokenizer.TT_EOF)
-                     && (currentToken.sval.compareTo(ALIAS) != 0));
+                     && (!currentToken.sval.equals(ALIAS)));
```

Comment:
Nit: Not your issue but you can also get rid of unneeded brackets () at lines 128-129, 144-145.